### ISSUE TITLE
Purpose?

### ISSUE THREAD

**Issue by ts** [mkwmms/straps#2]
What are we trying to accomplish with this? I see a few different goals/directions it could go but it might make sense to try and hash out what it should do at the end of the day. I've got an outline of basic things that I ran into after swapping my boot SSD this weekend, maybe I should put that up as a use/test case or something.

**Comment by mkwmms** [comment]
I agree, I feel like I need to narrow my focus/goals for this project. 
As far as I see it right now we could go two general directions:
- We define a simple framework (directory layout, naming etc.) with little to no actual scripts/apps. Something like [this](https://github.com/justinmayer/tacklebox) 
- We incorporate something like [reel](https://github.com/mkwmms/reel) (or not use it altogether) and then also essentially make this project a wrapper for mackup (or to be used alongside it) and ansible.
